ui/src/api/series/series.js

```diff
@@ -28,16 +33,73 @@ const update = async (seriesPid, data) => {
   return resp;
 };
 
-const createRelation = async (seriesPid, data) => {
-  const resp = await http.post(`${seriesURL}${seriesPid}/relations`, data);
+const createRelation = async (
+  referer,
+  selectedRelatedList,
+  relationType,
+  extraRelationField = {}
+) => {
+  let newRelationsPayload = [];
+  if (relationType === 'serial' || relationType === 'multipart_monograph') {
```

Comment:
what about having different methods instead of 3 ifs?

Might valid for doc too...